pages/positions/withdrawliquidity.tsx

```diff
@@ -125,14 +126,18 @@ const WithdrawLiquidity: NextPage = () => {
           </div>
         </div>
         <div className={style.buttonDiv}>
-          <div
-            className={style.confirmButton}
-            onClick={() => {
-              approveTransaction()
-            }}
-          >
-            Approve
-          </div>
+          {enableApprove ? (
+            <div
+              className={style.confirmButton}
+              onClick={() => {
+                approveTransaction()
+              }}
+            >
```

Comment:
you don't need all of these function declarations. Just do
```javascript
onClick={approveTransaction}
```
and it does the same operation

Reply:
Done